app/scripts/lib/accounts/BalancesController.ts

```diff
@@ -119,7 +124,7 @@ const balancesControllerMetadata = {
 
 const BTC_TESTNET_ASSETS = ['bip122:000000000933ea01ad0ee984209779ba/slip44:0'];
 const BTC_MAINNET_ASSETS = ['bip122:000000000019d6689c085ae165831e93/slip44:0'];
-export const BTC_AVG_BLOCK_TIME = 600000; // 10 minutes in milliseconds
+export const BTC_AVG_BLOCK_TIME = 10 * 60 * 1000; // 10 minutes in milliseconds
```

Comment:
WDYT about changing it to 5 min, so we are at most 10 min behind (1 block time)?